internal/provider/resource_access_flow.go

```diff
@@ -89,11 +89,8 @@ func (a accessFlowResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				Required:            true,
 				Attributes: map[string]schema.Attribute{
 					"type": schema.StringAttribute{
-						MarkdownDescription: "Type of trigger. Only `user_request` is supported.",
+						MarkdownDescription: "Type of trigger.`user_request` or 'auto_grant' is supported.",
 						Required:            true,
-						Validators: []validator.String{
-							stringvalidator.OneOf("user_request"),
-						},
```

Comment:
why no validator?

Reply:
because the API is validating this field and this allow the future versions to be more easy to add for them in the API.